sha/host/src/main.rs

```diff
@@ -1,7 +1,8 @@
 use clap::{Arg, Command};
 use methods::{HASH_ID, HASH_PATH};
 use risc0_zkp::core::sha::Digest;
-use risc0_zkvm::host::{Prover, Receipt};
+use risc0_zkvm::prove::Prover;
+use risc0_zkvm::receipt::Receipt;
```

Comment:
These are re-exported at the top level, would prefer `risc0_zkvm::{Prover, Receipt};`

Also happy to discuss pros/cons of each approach.